website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -185,13 +185,13 @@ __Behavioral Fields__
 
 To enable the drift feature flag, refer to the [Settings Feature Gates]({{<ref "./settings#feature-gates" >}}).
 
-Karpenter will annotate the nodes with the `karpenter.sh/voluntary-disruption: "drifted"` if the node is drifted, and does not have the annotation,
+Karpenter will add `MachineDrifted` status condition on the machines if the machine is drifted, and does not have the status condition,
```

Comment:
This will come out of date as soon as we refer to this as "NodeClaims". @jonathan-innis, are we gonna do a docs pass afterwards?